### PR TITLE
Issue#228 manage questions on polls

### PR DESCRIPTION
**Pull request by IRoss-git**
dev
## ZenHub

* [Main ZenHub ticket](https://app.zenhub.com/workspaces/home-project-5f7b77ff8db73a001cb17009)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Illia-Shchur — APPROVED** [review]
Looks fine.

File: home-open-api/src/main/resources/yaml/openapi.yaml [hunk 1]
```diff
@@ -2615,4 +2924,11 @@ components:
...
-      example: draft
\ No newline at end of file
+      example: draft
+    question_type:
+      in: query
+      name: type
+      schema:
+        $ref: '#/components/schemas/QuestionType'
+      description: Type of the Question
+      example: advice
```
Comment (kh0ma) [inline home-open-api/src/main/resources/yaml/openapi.yaml]: Can we add new line at the end of the file?

**Review by kh0ma — APPROVED** [review]

File: home-open-api/src/main/resources/yaml/openapi.yaml (hunk 1 above)
Comment (IRoss-git) [reply home-open-api/src/main/resources/yaml/openapi.yaml]: Sure, added

File: home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/PollQuestionServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+@RequiredArgsConstructor
+public class PollQuestionServiceImpl implements PollQuestionService {
+
+    private final PollQuestionRepository pollQuestionRepository;
+
+    private final PollRepository pollRepository;
+
+    private final ServiceMapper mapper;
+
+    private static final String POLL_WITH_ID_NOT_FOUND = "Poll with 'id: %d' is not found";
```
Comment (likeRewca) [inline home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/PollQuestionServiceImpl.java]: I suggest you use the constant NOT_FOUND_MESSAGE from the interface QueryableService, and build th string as String.format(NOT_FOUND_MESSAGE, "Poll", id). Haw you think? 
Next constant the same.

**Review by AriannaLiss — APPROVED** [review]

File: home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/PollQuestionServiceImpl.java (hunk 2 above)
Comment (IRoss-git) [reply home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/PollQuestionServiceImpl.java]: Done

**Review by veniamin-aveltsev — APPROVED** [review]

**Review by likeRewca — APPROVED** [review]

**Pull request merged**